view/frontend/web/js/checkout/shipping_method/show-myparcel-shipping-method.js

```diff
@@ -84,32 +84,28 @@ define(
             }, 1000);
         }
 
-        function checkOnlyShowMailbox() {
-            if (_getCcIsLocal() === false || window.mypa.data.mailbox.active === false) {
+        function checkOnlyShowRadioButton(type) {
+            if (_getCcIsLocal() === false || window.mypa.data[type].active === false) {
                 return false
             }
 
             return true;
         }
 
-        function checkOnlyShowDigitalStamp() {
-            if (_getCcIsLocal() === false || window.mypa.data.digital_stamp.active === false) {
-                return false;
-            }
+        function showDeliveryRadio(type) {
```

Comment:
misschien is `extraOption` of `extraMethod` beter dan `type`. Wat denk jij?

Reply:
Done